frontend/tests/utils/downloadUtils.ts

```diff
@@ -472,3 +421,51 @@ export async function getActualImage(
     clip: { x: startX, y: startY, width, height },
   });
 }
+export async function verifyPng(
+  dirPath: string,
+  tissues: string[]
+): Promise<void> {
+  const verificationPromises = tissues.map(async (tissue) => {
+    const inputImagePath = `${dirPath}/${tissue}.png`;
+    const outputImagePath = `${dirPath}/${tissue}_cropped.png`;
+    const expectedPng = `${dirPath}/${tissue}_actual.png`;
+
+    await performExtraction(inputImagePath, outputImagePath);
+    const verified = await compareImages(expectedPng, outputImagePath);
+
+    if (verified) {
+      console.log(`${tissue} PNG file is verified.`);
+    } else {
+      console.log(`${tissue} PNG file verification failed.`);
+    }
+  });
+
+  await Promise.all(verificationPromises);
+}
+async function performExtraction(
+  inputImagePath: string,
+  outputImagePath: string
+): Promise<void> {
+  const startHeight = 560;
```

Comment:
Ah awesome! So we have an offset to only compare the heat map portion of the downloaded PNG? Great solution! Thank you!

Reply:
@victorikeokoro6  Should we delete all the fixture PNGs? Thanks!